JustSaying/AwsTools/JustSayingConstants.cs

```diff
@@ -41,7 +41,7 @@ public static class JustSayingConstants
         /// <summary>
         /// Default message retention period on a queue in seconds
         /// </summary>
-        public static int DEFAULT_RETENTION_PERIOD = 60 * 10;    //10 minutes
+        public static int DEFAULT_RETENTION_PERIOD = 345600;    //4 days
```

Comment:
wouldn't be better to have something like **3600 * 24 * 4**?
The end result is the same but is easier to read.

Reply:
I want this to be exactly, precisely, 100% identical to what was committed (multiple times) before. All of these have been changed to the less-idiotic `TimeSpan` in v7, by the way.